src/pybind/mgr/dashboard/frontend/src/main.ts

```diff
@@ -11,3 +11,18 @@ if (environment.production) {
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
   .catch((err) => console.log(err));
+
+/*
+ * Temporary fix for Chrome 80 bug regarding reduce method
+ * source: https://bugs.chromium.org/p/chromium/issues/detail?id=1049982#c7
+ */
+(function() {
+  const arrayReduce = Array.prototype.reduce;
+  let callback;
+  Object.defineProperty(Array.prototype, 'reduce', {
+    value: function(cb: any, ...args: any[]) {
+      callback = cb;
+      return arrayReduce.call(this, callback, ...args);
+    }
+  });
+})();
```

Comment:
Can we run this only on the 'buggy' Chrome version? Even more, if it's a bug in Chrome v80, I don't think we should add any workaround yet, but just wait for the fix to appear (or at least for any Chromium devs to comment on the issue report).

Reply:
I added a verification for the chrome version and it will only be applied in v80+.

I'm OK with waiting a few more days, but eventually we need to merge this to be able to run e2e tests.